README.md

```diff
@@ -123,6 +127,44 @@ integrates with the Smartling translation platform.
     The callback receives the default description string, the job `TranslationSource` instance, and the list of
     target `Translation`s. It expected to return string.
 
+    ----
+
+    If you want to pass a [Visual Context](https://help.smartling.com/hc/en-us/articles/360057484273--Overview-of-Visual-Context)
+    to Smartling after a Job is synced, you need to provide a way to get hold
+    of the appropriate URL for the page to use context. You provide this via
+    the `VISUAL_CONTEXT_CALLBACK` setting.
+
+    If this callback is defined, it will be used to send the visual context to Smartling.
+    This step happens just after the regular sync of a Job to Smartling and _only_ if
+    the callback is defined.
+
+    The callback must take the Job instance and return:
+
+    1. a URL for the page that shows the content used to generate that Job
+    2. the HTML of the page.
+
+    ```python
+    from wagtail_localize.models import Job
+
+    def get_visual_context(job: Job) -> tuple[str, str]:
+
+        # This assumes the page is live and visible. If the page is a draft, you
+        # will need a some custom work to expose the draft version of the page
+        page = job.translation_source.get_source_instance()
+        page_url = page.url
```

Comment:
question: should this not be `page.full_url` as for single sites this can be relative.